f8a_report/s3_helper.py

```diff
@@ -18,8 +18,9 @@ def __init__(self):
         self.aws_s3_access_key = os.environ.get('AWS_S3_ACCESS_KEY_ID')
         self.aws_s3_secret_access_key = os.environ.get('AWS_S3_SECRET_ACCESS_KEY')
         self.deployment_prefix = os.environ.get('DEPLOYMENT_PREFIX') or 'dev'
-        self.report_bucket_name = os.environ.get('REPORT_BUCKET_NAME')
-
+        self.deployment_prefix = self.deployment_prefix.lower()
```

Comment:
Or you can combine them into one
```
self.deployment_prefix = os.environ.get('DEPLOYMENT_PREFIX').lower() or 'dev'
```

Reply:
If I will combine them the tests will fail. Because `os.environ.get('DEPLOYMENT_PREFIX')` returns `null` in the case where it didn't find the deployment prefix and `null` type has no function `lower`. Check my previous commit.